flintrock/ec2.py

```diff
@@ -79,25 +80,43 @@ def instances(self):
         else:
             return self.slave_instances
 
+    @property
+    def private_net(self):
+        # check private mode
```

Comment:
I'd delete this comment. It's unnecessary.

Reply:
Ok